safe/definitions/styles.py

```diff
@@ -20,9 +20,9 @@
 very_dark_red = QColor('#710017')
 MMI_white = QColor('#ffffff')
 MMI_blue = QColor('#209fff')
-MMI_light_blue = QColo ('#00cfff')
+MMI_light_blue = QColor('#00cfff')
```

Comment:
I wonder if it would not be a better idea to name these by level 

```
MMI_0 = QColor('#ffffff')
```

I think it would carry more implicit meaning than colour names. For H/M/L legends we could make verbal aliases e.g.:

```
MMI_high = MMI_red
```

Anyway that is just a thought and certainly not a blocker to merging. LGTM